### PR TITLE
Clippy, rustfmt, Cargo metadata updates

### PR DESCRIPTION
**Pull request by urschrei**
- [x] I agree to follow the project's [code of conduct](https://github.com/georust/geo/blob/master/CODE_OF_CONDUCT.md).
- [x] I added an entry to `CHANGES.md` if knowledge of this change could be valuable to users.
---

### CODE REVIEW

File: rstar/Cargo.toml [hunk 1]
```diff
@@ -1,19 +1,17 @@
 [package]
 name = "rstar"
-version = "0.8.3"
-authors = ["Stefan Altmayer <stoeoef@gmail.com>"]
+version = "0.8.4"
```
Comment (michaelkirk) [inline rstar/Cargo.toml:3]: No big deal, but I usually don't bump the version until just before the release;

1. we don't know what other (breaking) changes might go in before release
2. less to keep track of so you don't have to double-check every time you make a PR if you should be bumping the version

Is there a benefit to bumping now?

**Review by michaelkirk — APPROVED** [review]
LGTM - just some "take it or leave it" feedback.

File: rstar/Cargo.toml (hunk 1 above)
Comment (urschrei) [reply rstar/Cargo.toml:3]: No, it was purely a "we'll be bumping anyway" thing. Will revert.

**Comment by urschrei** [comment]
bors r=michaelkirk

**Pull request merged**